pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name = "wagtail-org"
+version = "2.0.0"
+description = "The wagtail.org Wagtail site"
+authors = ["Torchbox <hello@torchbox.com>"]
+license = "BSD-3-Clause"
+readme = "README.md"
+packages = [{include = "wagtailio"}]
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

Comment:
question: any reason not to make the leap to 3.9 if not 3.10?

Reply:
No, done.

Reply:
_record scratch_ Actually, the ancient version of `fab` that we're using breaks on Python>=3.9 due to `collections.abc.Mapping` not being importable from `collections` any more, so I've reverted that change and we're still on 3.8 for now.